### ISSUE TITLE
Controller unit test error on  PHP 7.2

### ISSUE THREAD

**Issue by tuanpht** [spiral/application#26]
I get erros when run unit test.

My environment:
- Ubuntu 18.04
- PHP 7.2.10 

The error:
```
1) Tests\Controllers\IndexTest::testSeeWelcome
ini_set(): Headers already sent. You cannot change the session module's ini settings at this time

/spiral-demo/vendor/spiral/framework/source/Spiral/Session/SessionFactory.php:58
/spiral-demo/vendor/spiral/framework/source/Spiral/Session/Http/SessionStarter.php:79
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:128
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:227
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/Middlewares/CsrfMiddleware.php:74
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:128
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:227
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/Middlewares/ExceptionWrapper.php:69
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:128
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:227
/spiral-demo/vendor/spiral/profiler/source/Profiler/ProfilerWrapper.php:96
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:128
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/MiddlewarePipeline.php:97
/spiral-demo/vendor/spiral/framework/source/Spiral/Http/HttpCore.php:119
/spiral-demo/tests/HttpTest.php:40
/spiral-demo/tests/Controllers/IndexTest.php:16
```

I debug with `headers_sent()`, and find that headers are sent at:
```
/spiral-demo/vendor/phpunit/phpunit/src/Util/Printer.php:112
```
Actually, it prints PHPUnit information.

**Comment by wolfy-j** [comment]
There is some behavior change of how PHP7.2 handle sessions in CLI mode, it's easy to fix - point PHPUnit output to STDERR:

https://github.com/spiral/session/blob/master/phpunit.xml#L13

We have found and fixed the same issue for 2.0 release.

**Comment by tuanpht** [comment]
@wolfy-j Thank you very much.

Do you plan when to release 2.0?

**Comment by wolfy-j** [comment]
By December of this year, we plan to incorporate all of the internal work done in a last 2 years: 40 times higher performance (integrated https://github.com/spiral/roadrunner), async jobs and a lot of new stuff. You can take a look at the current state of the core here: https://github.com/spiral/framework/tree/2.0

PHPUnit issue has been fixed in 1.0.2

**Comment by wolfy-j** [comment]
Though, current 1.0 works perfectly with RoadRunner on production for the last 4 months.
